presto-main/src/test/java/com/facebook/presto/sql/planner/TestLocalExecutionPlanner.java

```diff
@@ -46,7 +46,7 @@ public void cleanup()
         runner = null;
     }
 
-    @Test
+    @Test(enabled = false)
```

Comment:
It's pretty much equal to removing this test. No one will remember about it being disabled after some time...